Survey Checks/README.md

```diff
@@ -0,0 +1,32 @@
+# Survey checks
+
+Here we provide detailed instructions for preparing your electronic survey with the aim of adding survey checks and creating a real-time high-frequency check dashboard.
+
+1. [Prepare Survey forms](https://github.com/dime-worldbank/iesurveykit/blob/initial-update/Survey%20Checks/1-prepare-scto-forms.md)
+2. [Adapt Survey forms](https://github.com/dime-worldbank/iesurveykit/blob/initial-update/Survey%20Checks/2-adapt-scto-forms.md)
+3. [Set up HFC Dashboard](https://github.com/dime-worldbank/iesurveykit/blob/initial-update/Survey%20Checks/3-set-up-hfc-dashboard.md)
+4. [Complementary analysis](https://github.com/dime-worldbank/iesurveykit/blob/initial-update/Survey%20Checks/4-complementary-analysis-r.md)
+5. [Add survey completion checks](https://github.com/dime-worldbank/iesurveykit/blob/initial-update/Survey%20Checks/5-survey-completion-checks.md)
+
+
+We also outline some [best practices and commonly faced issues](https://github.com/dime-worldbank/iesurveykit/blob/initial-update/Survey%20Checks/best-practices-and-issues.md) in setting up these real time HFC dashboards.
+
+## Relevant Links
```

Comment:
I'm a little unsure whether it's useful to have these in the main readme, vs in the relevant subsections